core/tools/blueos_startup_update/blueos_startup_update

```diff
@@ -71,6 +72,32 @@ cmdline_file = None
 import collections
 from commonwealth.utils.commands import run_command
 
+class CpuType(str, Enum):
+    PI4 = "Raspberry Pi 4 (BCM2711)"
+    PI5 = "Raspberry Pi 5 (BCM2711)"
+    other = "other"
+
+class HostOs(str, Enum):
+    Bookworm = "Debian(Raspberry Pi OS?) 12 (Bookworm)"
+    Bullseye = "Debian(Raspberry Pi OS?) 11 (Bullseye)"
+
+def get_cpu_type() -> CpuType:
+    with open('/proc/cpuinfo', 'r') as f:
+        for line in f:
+            if 'Raspberry Pi 4' in line:
+                return CpuType.PI4
+            if 'Raspberry Pi 5' in line:
+                return CpuType.PI5
+    return CpuType.other
+
+def get_host_os() -> HostOs:
+    os_release = load_file('/etc/os-release')
+    if 'bookworm' in os_release:
+        return HostOs.Bookworm
+    if 'bullseye' in os_release:
+        return HostOs.Bullseye
+    return HostOs.other
```

Comment:
There is no definition for other, where are the linters ? :eyes:

Reply:
it is not working for blueos_startup_update because it doesn't have the .py extension >.<
even manually,.
let me see what I can do

Reply:
https://github.com/bluerobotics/BlueOS/pull/2645